src/net/sourceforge/kolmafia/AdventureResult.java

```diff
@@ -640,6 +640,17 @@ public static AdventureResult parseResult(final String s) {
   }
 
   public static AdventureResult parseItem(final String s, final boolean pseudoAllowed) {
+    // Certain items fountain parentheses. Appending (COUNT) to such is problematic.
```

Comment:
"fountain" ?  If that's not a typo then I don't understand the idiom :-)  Not blocking.

Reply:
Contain.